apis/Google.Bigquery.V2/Google.Bigquery.V2/BigqueryClient.JobCrud.cs

```diff
@@ -149,5 +151,153 @@ public virtual BigqueryJob CancelJob(JobReference jobReference, CancelJobOptions
+        /// Asynchronously lists the jobs within this client's project.
+        /// This method just creates a <see cref="ProjectReference"/> and delegates to <see cref="ListJobsAsync(ProjectReference, ListJobsOptions)"/>.
+        /// </summary>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>An asynchronous sequence of the jobs within this project.</returns>
+        public virtual IPagedAsyncEnumerable<JobList, BigqueryJob> ListJobsAsync(ListJobsOptions options = null) =>
+            ListJobsAsync(GetProjectReference(ProjectId), options);
+
+        /// <summary>
+        /// Asynchronously lists the jobs within the specified project.
+        /// This method just creates a <see cref="ProjectReference"/> and delegates to <see cref="ListJobsAsync(ProjectReference, ListJobsOptions)"/>.
+        /// </summary>
+        /// <param name="projectId">The project to list the jobs from. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>An asynchronous sequence of the jobs within the specified project.</returns>
+        public virtual IPagedAsyncEnumerable<JobList, BigqueryJob> ListJobsAsync(string projectId, ListJobsOptions options = null) =>
+            ListJobsAsync(GetProjectReference(projectId), options);
+
+        /// <summary>
+        /// Asynchronously lists the jobs within the specified project.
+        /// This method just creates a <see cref="ProjectReference"/> and delegates to <see cref="ListDatasetsAsync(ProjectReference, ListDatasetsOptions)"/>.
+        /// </summary>
+        /// <param name="projectReference">A fully-qualified identifier for the project. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>An asynchronous sequence of the jobs within the specified project.</returns>
+        public virtual IPagedAsyncEnumerable<JobList, BigqueryJob> ListJobsAsync(ProjectReference projectReference, ListJobsOptions options = null)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Polls the job with the specified ID in this client's project for completion.
+        /// This method just creates a <see cref="JobReference"/> and delegates to <see cref="PollJobUntilCompletedAsync(JobReference,GetJobOptions,PollSettings,CancellationToken)"/>.
+        /// </summary>
+        /// <param name="jobId">The job ID. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="pollSettings">The settings to control how often and long the job is fetched before timing out if it is still incomplete.</param>
```

Comment:
Other parameter descriptions indicate whether null is allowed or not. But the pollSettings parameter descriptions do not. Is this intentional?